cli/cmd/project/edit.go

```diff
@@ -49,66 +36,47 @@ func EditCmd(ch *cmdutil.Helper) *cobra.Command {
 				OrganizationName: ch.Org,
 				Name:             name,
 			}
-			promptFlagValues := ch.Interactive
+
+			// Set values from flags if they were changed
+			if cmd.Flags().Changed("prod-ttl-seconds") && prodTTL < 0 {
+				return fmt.Errorf("prod-ttl-seconds must be non-negative")
+			}
```

Comment:
Since it's rare this field is changed, this kind of validation is nicer to do once in the backend than in the client.

Probably in the database model layer, around here (look for the `validate:` struct tags elsewhere in the file for examples of how to add validation): https://github.com/rilldata/rill/blob/a4ced7d46771430003c26ba3f3fd21812c875865/admin/database/database.go#L446